AttitudeManager/Inc/AM.hpp

```diff
@@ -12,19 +12,32 @@
 #include "AM_ControlAlgorithm.hpp"
 #include "CommonDataTypes.hpp"
 
+#include <vector>
+
 namespace AM {
 
 class AttitudeManager {
    public:
-    AttitudeManager(Flightmode* control_algorithm) : control_algorithm(control_algorithm){};
+    AttitudeManager(Flightmode* control_algorithm);
 
     void runControlLoopIteration(const AttitudeManagerInput& instructions);
 
    private:
     AttitudeManager();
+    void outputToMotor(config::ControlAxis_t axis, uint8_t percent);
 
-    Flightmode* control_algorithm;
+    Flightmode* control_algorithm_;
+    MotorInstance_t motorInstances_[config::NUM_MOTORS]{};
+    MotorInstance_t *motorReferences_[5]{nullptr};
+    
 };
+
+// !!unsure if the struct definition should be here, would appreciate feedback
+typedef struct {
+        MotorChannel *motorInstance;
+        bool isInverted;
+    } MotorInstance_t;
+
 }  // namespace AM
```

Comment:
I would suggest either placing this as a member struct (and dropping the typedef) of AttitudeManager, or moving it to Common. I doubt it will be used by any other code, so leaving it here is probably best.